lib/DDG/Spice/SalesTax.pm

```diff
@@ -4,41 +4,38 @@ package DDG::Spice::SalesTax;
 use strict;
 use DDG::Spice;
 use Locale::SubCountry;
-use YAML::XS 'LoadFile';
+#use YAML::XS 'LoadFile';
```

Comment:
Lets lose any commented-out code. You can `git add --patch` to make sure you don't include any comments in your commits (use `git help add` to find out more).